src/inventory_ui.h

```diff
@@ -556,6 +560,7 @@ class inventory_selector
         inventory_column map_column;         // Column for map and vehicle items
 
         const int border = 1;                // Width of the window border
+        std::string filter = "";
```

Comment:
This initialization is pointless - the (implicitly called) default constructor will do the same.